### PR TITLE
Rendering check and Meaningful Words (#Issues 1 and 8)

### PR DESCRIPTION
**Pull request by man1han**
#1 
I started with this because I don't really know much of React.
#8 
I did this one because I was interested in playing around more. Might try to do the others in my free time.

### CODE REVIEW

File: challenge/src/strings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "HackRU Reactment Challenge",
-    "greeting": "Hello World!",
+    "greeting": "Reactment is not HTML. #Im19andThisIsDeep.",
```
Comment (mjrb) [inline challenge/src/strings.json]: lol

File: challenge/src/App.jsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ class App extends Component {
     }
     render() {
         return (
+            console.log("Redered successfully! I think?"),
```
Comment (mjrb) [inline challenge/src/App.jsx:16]: @man1han this looks a little wonky to me...
why are you returning the result of console.log with the rendered function

**Review by mjrb — CHANGES_REQUESTED** [review]
perhaps we shouldn't be returning the result of console.log

**Comment by man1han** [comment]
Since the issue #8 says to add a log statement in the render method... I
found the render() function/method and added a log statement inside it. So
everytime render function is called there will be a statement in the log?
I'm sorry if I did something wrong, please let me know if I misunderstood
something.

Thankyou.

-Manthan

**Comment by mjrb** [comment]
@man1han what you put technically works, it's just I would have put the console.log before the return call in between lines 12 and 13. Mostly because console.log doesn't return anything and it's a bit more straight forward if the only thing the render function is returning is jsx. I'm actually kinda surprised this worked.
If you make that change I'll merge it in today

**Review by mjrb — APPROVED** [review]

**Pull request merged**